src/pages/subscription/subscription.ts

```diff
@@ -46,4 +55,25 @@ export class SubscriptionPage {
   ionViewCanEnter(): boolean | Promise<any> {
     return this.user.isAuthenticated(this.navCtrl);
   }
+
+  onOfferClick(offer): void {
+
+    const handler = StripeCheckout.configure({
+      key: STRIPE_PUBLISHABLE_API_KEY,
+      image: 'https://goo.gl/EJJYq8', // TODO: Update image shown in payment dialog
+      locale: this.translate.getBrowserLang(),
+      token: token => {
+        this.helpers.displayMessage(this.translationValues.STRIPE_SUCCESS_MESSAGE)
```

Comment:
this token we need to store, as that is what we will use to actually charge the user ?

Reply:
https://stripe.com/docs/quickstart#using-payment-information